assets/queries/openAPI/general/api_key_exposed_in_global_security/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "aecee30b-8ea1-4776-a99c-d6d600f0862f",
+  "queryName": "API Key Exposed In Global Security (v3)",
+  "severity": "MEDIUM",
+  "category": "Access Control",
+  "descriptionText": "API Keys should not be transported over network",
+  "descriptionUrl": "https://swagger.io/specification/#security-scheme-object",
+  "platform": "OpenAPI",
+  "override": {
+    "2.0": {
+      "id": "533a0d13-6e89-4551-ae33-bce14e5849c1",
+      "queryName": "API Key Exposed In Global Security (v2)",
+      "descriptionUrl": "https://swagger.io/specification/v2/#securitySchemeObject"
```

Comment:
I think the link should be: https://swagger.io/specification/v2/#securityDefinitionsObject